demos/ota/aws_iot_ota_update_demo.c

```diff
@@ -266,10 +266,6 @@ static int _establishMqttConnection( bool awsIotMqttMode,
     networkInfo.pNetworkInterface = pNetworkInterface;
     networkInfo.disconnectCallback.function = prvNetworkDisconnectCallback;
 
-    #if ( IOT_MQTT_ENABLE_SERIALIZER_OVERRIDES == 1 ) && defined( IOT_DEMO_MQTT_SERIALIZER )
```

Comment:
Is this config (and `IOT_DEMO_MQTT_SERIALIZER ` function) not going to be applicable for the MQTT library anymore?

Reply:
Yes. That is correct